### PR TITLE
Bump Fragment to v1.3.6

### PR DESCRIPTION
**Pull request by msasikanth**
- Bump Fragment to v1.3.6
- Disable new `FragmentStateManager`
- Update CHANGELOG

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,8 @@
 - Bump AppCompat to v1.3.0
 - Move online patient lookup behind a feature flag
 - Cache build dependencies and intermediates in CI workflows
+- Bump Fragment to v1.3.5
```
Comment (Honey14) [inline CHANGELOG.md:22]: We're switching to 1.3.6 stable release. Can you update to change to reflect the same?

File: app/src/main/java/org/simple/clinic/ClinicApp.kt [hunk 2]
```diff
@@ -43,13 +44,15 @@ abstract class ClinicApp : Application(), CameraXConfig.Provider {
 
   protected open val crashReporterSinks = emptyList<CrashReporter.Sink>()
 
-  @SuppressLint("RestrictedApi")
+  @SuppressLint("RestrictedApi", "UnsafeOptInUsageWarning")
```
Comment (Honey14) [inline app/src/main/java/org/simple/clinic/ClinicApp.kt:47]: Where can I find the definition of `UnsafeOptInUsageWarning` id that we're asking lint to ignore?

**Review by Honey14 — CHANGES_REQUESTED** [review]
LGTM. Left changes. PTAL

File: CHANGELOG.md (hunk 1 above)
Comment (msasikanth) [reply CHANGELOG.md]: Ah, I thought I made this change. Good catch 👍🏾

File: app/src/main/java/org/simple/clinic/ClinicApp.kt (hunk 2 above)
Comment (msasikanth) [reply app/src/main/java/org/simple/clinic/ClinicApp.kt:47]: http://googlesamples.github.io/android-custom-lint-rules/checks/UnsafeOptInUsageWarning.md.html

**Review by Honey14 — APPROVED** [review]
LGTM

**Pull request merged**